docs/how-to/how-to-run-kata-containers-with-SNP-VMs.md

```diff
@@ -44,12 +44,10 @@ $ popd
 - Build a custom QEMU
 ```bash
 $ source kata-containers/tools/packaging/scripts/lib.sh
-$ qemu_url="$(get_from_kata_deps "assets.hypervisor.qemu.snp.url")"
-$ qemu_branch="$(get_from_kata_deps "assets.hypervisor.qemu.snp.branch")"
-$ qemu_commit="$(get_from_kata_deps "assets.hypervisor.qemu.snp.commit")"
-$ git clone -b "${qemu_branch}" "${qemu_url}"
+$ qemu_url="$(get_from_kata_deps "assets.hypervisor.qemu-snp-experimental.url")"
+$ qemu_tag="$(get_from_kata_deps "assets.hypervisor.qemu-snp-experimental.tag")"
+$ git clone -b "${qemu_tag}" "${qemu_url}"
 $ pushd qemu
-$ git checkout "${qemu_commit}"
```

Comment:
Don't we want to keep a specific qemu commit, or do we want to trust AMD to not update the snp-v3 branch in a way that breaks us?

Reply:
The versions file now has a tag in it, which we point to above when we do the clone, so no need to do a checkout here. The tag should always point to the same commit.